src/command/output.rs

```diff
@@ -171,51 +172,156 @@ impl Convertable<Output> for BigEndianLength<Output> {
     }
 }
 
-#[test]
-fn test_invalid_length() {
+#[cfg(test)]
+mod tests {
+    use super::*;
     use crate::ArtCommand;
 
-    let command = ArtCommand::Output(Output {
-        data: vec![0xff; 512].into(),
-        ..Output::default()
-    });
-    let buffer = command.into_buffer().unwrap();
-    // #6: length needs to be encoded in big endian
-    assert_eq!(&buffer[0x10..=0x11], &[2, 0]);
+    #[test]
+    fn create_single_dmx_value_art_dmx_packet() {
+        let command = ArtCommand::Output(Output {
+            data: vec![255].into(), // The data we're sending to the node
+            ..Output::default()
+        });
+        let bytes = command.into_buffer().unwrap();
+        let comparison = vec![
+            65, 114, 116, 45, 78, 101, 116, 0, 0, 80, 0, 14, 0, 0, 1, 0, 0, 2, 255, 0,
+        ]; //is padded with zero to even length of two
+        assert!(bytes == comparison)
```

Comment:
For all `assert!(a == b)`, it is better to use `assert_eq!(a, b)`. That way Rust will tell you that `<left> does not match <right>` instead of `assertion failed`